wiki/Tournaments/CIC/2018/id.md

```diff
@@ -0,0 +1,157 @@
+  - [Rabpit - Sacred (ursa) \[Platter\]](https://osu.ppy.sh/beatmapsets/92456/#fruits/341590)
+  - [Camellia feat. Nanahira - Seashore on the moon- (Mir) \[LMT's Hard\]](https://osu.ppy.sh/beatmapsets/712171/#fruits/1519257)
+- Hidden
+  - [MDK - Fingerbang (CookieBite) \[Hyperion's Rain\]](https://osu.ppy.sh/beatmapsets/559525/#fruits/1242851)
+  - [Erosion - Shinkai Shoujo-piano ver.-(celerih) \[Deep Sea Rain\]](https://osu.ppy.sh/beatmapsets/634939/#fruits/1347079)
+  - [Shimotsuki Haruka - Kaleidoscope (Shurelia) \[Insane\]](https://osu.ppy.sh/beatmapsets/169622/#fruits/410798)
+- Tiebreaker
+  - [**\*namirin - Iscariot no Sakebi (Hareimu) \[Treason\]**](https://osu.ppy.sh/beatmapsets/685542/#fruits/1451063)
+
+## Peraturan
+
+### Sistem Turnamen
+
+1. Turnamen akan menggunakan sistem skoring **ScoreV2.**
+2. Pemilih map akan memberikan mappool untuk pertandingan maksimal seminggu sebelum pertandingan dimulai.
+3. Jadwal akan ditentukan oleh staf, jika mengalami masalah dengan jadwal, silahkan hubungi staf untuk permohonan perubahan jadwal.
+4. Peserta akan diundang ke dalam pertandingan multiplayer 15 menit sebelum pertandingan dimulai.
+5. Jika peserta belum hadir pada waktu yang telah ditentukan, maka pertandingan akan diundur maksimal 10 menit.
+6. Jika wasit atau staff terkait tidak ada yang aktif, maka pertandingan akan diundur.
+7. Jika saat pertandingan sedang berlangsung dan peserta gagal menyelesaikan map, maka peserta tersebut akan dinyatakan kalah dan masuk di peringkat terakhir.
+  - Pengecualian jika peserta bisa revive, maka peringkat akan ditentukan berdasarkan skor terbesar.
+8. Jika peserta mengalami permasalahan atau disconnect maka akan dihitung sebagai failed.
+  - Jika peserta disconnect sebelum 30 detik pertama setelah map dimainkan maka pertandingan boleh diulang (rematch).
+9. Jika peserta mengalami permasalahan atau disconnect di tengah pertandingan maka pertandingan akan diberhentikan sementara maksimal 15 menit.
+10. Lag bukan menjadi alasan untuk memberhentikan pertandingan yang sedang berlangsung.
+11. Jika ada permasalahan di dalam pertandingan, maka host/co-host akan membuat keputusan berdasarkan laporan yang didapat dari wasit.
+12. Diharapkan untuk menghormati satu sama lain dan bersikap sopan ketika bertanding. Penalti akan diberikan jika ada yang melanggar.
+  - Jika peserta memancing perkelahian dengan peserta lain, peserta yang memulai akan dilaporkan ke staf CIC dan diblacklist di instalasi selanjutnya.
+  - Penggunaan program ilegal akan dilaporkan ke staf osu! dan diblacklist di instalasi selanjutnya.
+  - Membantu peserta (joki) akan diblacklist akun untuk turnamen yang akan datang baik peserta maupun penjoki.
+
+### Sistem Babak Grup
+
+1. Setiap grup akan diisi oleh 3 peserta.
+2. Peserta akan bertanding dengan sistem Battle Royale.
+3. Babak grup akan menggunakan poin untuk setiap peserta dengan ketentuan sebagai berikut:
+  - Juara 1 : 2 Poin
```

Comment:
Spacing between `1` and `:`
It should be `Juara 1: 2 Poin.`

Apply to other instances:
Juara 2: 1 Poin.
Juara 3: -1 Poin.
Juara 4: -2 Poin.